Detectors/GlobalTracking/src/MatchTOF.cxx

```diff
@@ -352,6 +356,16 @@ bool MatchTOF::prepareTPCData()
     if constexpr (isTRDTrack<decltype(trk)>()) {
       this->addTRDSeed(trk, gid, time0, terr);
     }
+    if constexpr (isITSTrack<decltype(trk)>()) {
+      //      float tMin = nBC * o2::constants::lhc::LHCBunchSpacingMUS + ITSTimeBiasMUS;
+      //      float tMax = (nBC + mITSROFrameLengthInBC) * o2::constants::lhc::LHCBunchSpacingMUS + ITSTimeBiasMUS;
+
+      //      time0 += itsBias;
+      //      terr *= this->mITSROFrameLengthMUS;               // error is supplied as a half-ROF duration, convert to \mus
+      terr = 5; // just to start
```

Comment:
this will not work for PbPb, there the ROF length can be >20 \mus. Why not taking the time estimated based on ROF length?

Reply:
Here my doubt is the following. Is the case to set the error at a general level and then provide it via CreateVariadic instead that managing it in the TOF scope? For instance iis also TRD would do the same it is in common.
If you prefer I can implement it here. Let me know

Reply:
If the question is why the CreateVariadic does not provide a final error in \mus, then it is because I don't want it to make dependent on the CCDB classes and the CCDB fetcher: setting up the access to CCDB is up to the workflow rather than data container helper.

Reply:
Ok, I will implement it in the TOF matcher